datafusion/optimizer/tests/integration-test.rs

```diff
@@ -87,11 +88,11 @@ fn case_when_aggregate() -> Result<()> {
 
 #[test]
 fn unsigned_target_type() -> Result<()> {
-    let sql = "SELECT * FROM test WHERE col_uint32 > 0";
+    let sql = "SELECT col_utf8 FROM test WHERE col_uint32 > 0";
```

Comment:
As I added some columns to the test data I changed these plans so they selected specific columns rather than `SELECT *` so they didn't diff when the schema changed

Reply:
clever 🤣

Reply:
"lazy" would also be a good description :)